content/news/2023-05-25-maplibre-gl-js-v3/index.md

```diff
@@ -13,7 +13,7 @@ Most users will be able to update to v3 without code changes, but it's recommend
 
 Thanks for the code contributions!
 
-[@VehpuS](https://github.com/VehpuS), [@ZeLonewolf](https://github.com/ZeLonewolf), [@kevinschaul](https://github.com/kevinschaul), [@zstadler](https://github.com/zstadler), [@smellyshovel](https://github.com/smellyshovel), [@jleedev](https://github.com/jleedev), [@kircher1](https://github.com/kircher1), [@prozessor13](https://github.com/prozessor13), [@llambanna](https://github.com/llambanna), [@IvanSanchez](https://github.com/IvanSanchez), [@tordans](https://github.com/tordans), [@ChrisLoer](https://github.com/ChrisLoer), [@Pessimistress](https://github.com/Pessimistress), [@cns](https://github.com/cns), [@acalcutt](https://github.com/acalcutt), [@vanilla](https://github.com/vanilla), [@lukashass](https://github.com/lukashass), [@birkskyum](https://github.com/birkskyum), [@HarelM](https://github.com/HarelM), [@zhangyiatmicrosoft](https://github.com/zhangyiatmicrosoft), [@rotu](https://github.com/rotu), [@pramilk](https://github.com/pramilk), [@wipfli](https://github.com/wipfli), [@xabbu42](https://github.com/xabbu42)
+[@prozessor13](https://github.com/prozessor13), [@rotu](https://github.com/rotu), [@acalcutt](https://github.com/acalcutt), [@ambientlight](https://github.com/ambientlight), [@Pessimistress](https://github.com/Pessimistress), [@HarelM](https://github.com/HarelM), [@birkskyum](https://github.com/birkskyum), [@zhangyiatmicrosoft](https://github.com/zhangyiatmicrosoft), [@pramilk](https://github.com/pramilk), [@wipfli](https://github.com/wipfli), [@xabbu42](https://github.com/xabbu42), [@ZeLonewolf](https://github.com/ZeLonewolf), [@kevinschaul](https://github.com/kevinschaul), [@zstadler](https://github.com/zstadler), [@smellyshovel](https://github.com/smellyshovel), [@jleedev](https://github.com/jleedev), [@kircher1](https://github.com/kircher1), [@llambanna](https://github.com/llambanna), [@IvanSanchez](https://github.com/IvanSanchez), [@tordans](https://github.com/tordans), [@ChrisLoer](https://github.com/ChrisLoer), [@cns](https://github.com/cns), [@vanilla](https://github.com/vanilla), [@lukashass](https://github.com/lukashass), [@VehpuS](https://github.com/VehpuS), 
```

Comment:
Should be @cns-solutions-admin maybe?

Reply:
Oh yes, the dash tripped me - same with vanilla --> vanilla-lake I see